packages/os/os.spec

```diff
@@ -488,9 +489,9 @@ install -p -m 0644 %{S:121} %{buildroot}%{_cross_unitdir}
 %{_cross_attribution_vendor_dir}
 
 %if %{with systemd_networkd}
-%dir %{_cross_unitdir}/generate-network-config.service.d
-%{_cross_unitdir}/generate-network-config.service.d/requires-mounts-network-config.conf
-%{_cross_unitdir}/etc-systemd-network.mount
+%{_cross_unitdir}/write-network-status.service
+%dir %{_cross_libdir}/systemd/resolved.conf.d
+%{_cross_libdir}/systemd/resolved.conf.d/00-resolved.conf
```

Comment:
Can we put these in the `netdog` sub-package instead? It's weird to use the otherwise mostly-empty `os` package for this.